backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3ApplicationConfiguration.java

```diff
@@ -185,6 +191,18 @@ public void setForegroundFPS (int fps) {
 		this.foregroundFPS = fps;
 	}
 
+	/** Sets whether to render and fire {@link LifecycleListener#pause()}/{@link LifecycleListener#resume()} events on when window
+	 * is minimized/restored. **/
```

Comment:
That's not entirely true. `#render()` is still called even though the application is paused.

Reply:
What I do when making a desktop game is `Gdx.graphics.setContinuousRendering(false)` on pause and `true` on resume, as well as pausing any music, stopping or muting any sounds. This might not work for all games, e.g. if they have networking that relies on the game loop running, but it does feel like one of those things that shouldn't be my own responsibility to handle.

Reply:
I see. To me not rendering while minimized shuld be the default behaviour even if there was a mechanism to disable it for those special cases you mention.

In any case, shall it say `/** Sets whether to pause the application {ApplicationListener@pause()}...` instead for example?

Reply:
Javadoc has been changed